app/views/user_list_uploads/invitation_attempts/index.html.erb

```diff
@@ -54,7 +54,7 @@
           </div>
           <div>
             <% if @all_invitations_attempted %>
-              <%= link_to "Terminer", @user_list_upload.structure_users_path, class: "btn btn-primary d-block mx-auto", data: { turbo_frame: "_top" } %>
+              <%= link_to "Terminer", @user_list_upload.structure_users_path, class: "btn btn-primary d-block mx-auto", data: { turbo_frame: "_top", controller: "tally", "tally-form-id": "nWjZGj", "tally-delay-in-ms": 1000, action: "click->tally#showPopup" } %>
```

Comment:
Il vaudrait pas mieux mettre l'id dans une variable d'env ? Ça permettrait de l'activer qu'en prod d'ailleurs

Reply:
Alors je l'ai pas mis dans une variable d'env car c'est vraiment lié à ce formulaire en particulier pas à Tally au global, et je me disais qu'on pourrait potentiellement en avoir 3-4 en même temps, rendant un peu pénible l'usage d'une variable d'env. 
Mais dans les faits on en aura surement qu'un ou 2 donc why not

Reply:
Pour le coup je pense que ce n'est pas une bonne idée de le laisser en dur, ne serait-ce que pour ne pouvoir l'afficher dans tous les env. On veut pas l'avoir en staging et en demo non ?